tests/Event/ReadModel/RDF/EventJsonToTurtleConverterTest.php

```diff
@@ -265,6 +265,7 @@ public function it_converts_an_event_with_periodic_calendar(): void
 
         $turtle = $this->eventJsonToTurtleConverter->convert($this->eventId);
 
+        file_put_contents(__DIR__ . '/ttl/event-with-calendar-periodic.ttl', $turtle);
```

Comment:
Are this all debug statements to get the new format?

Reply:
Yes! Sorry, should never have been committed.